### ISSUE TITLE
[Screen Reader-Domains-Home]: In browse mode, NVDA is not announcing position of menu items (Add domain, Buy domain..etc)

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13199]
### Steps to reproduce:
Step 1: Start NVDA
Step 2: Open the above URL in Edge dev browser.
Step 3: In browse mode, navigate to "Add domain, Buy domain..etc"
Step 4: Observe the behavior of screen reader when focus is on "Add domain, Buy domain..etc".

### Actual behavior:
In browse mode, NVDA does not announcing position of menu items (Add domain, Buy domain..etc)

### Expected behavior:
In browse mode, NVDA should announce position of menu items (Add domain, Buy domain..etc)
Ex: It should announce as "Add domain menu item 1 of 6"

### Additional Notes:
Issue repro only with NVDA in all browsers.

### System configuration:
Application: M365 Admin Portal
URL: https://admin.microsoft.com/Adminportal/Home
Browser: Edge, Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.1

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer?
Yes

**Comment by feerrenrut** [comment]
This needs further investigation by Microsoft developers. Please provide a minimal test case for us to investigate. I'm closing this issue for now.

**Comment by msftedad** [comment]
Hi @feerrenrut, we are able to reproduce this issue for menuitems through latest NVDA version 2021.3.1 on URL: https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html. NVDA is not announcing the position of menu item in browse mode. **Could you please reopen this bug?**

Repro Step:
1. Turn on NVDA.
2. Open URL: https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html in Edge browser.
3. Naviagte to the menu items in browse mode using down arrow key.
4. Observe that NVDA is not announcing position.

Screenshot:
![image](https://user-images.githubusercontent.com/93735775/153658140-4cbb8338-b557-4a1b-ade2-d71ead345b0d.png)


Actual:
NVDA is not announcing the position of menu items in browse mode navigation.

Expected:
NVDA should announce the position of menu items in browse mode navigation.

**Comment by feerrenrut** [comment]
I don't think it makes sense to interact with this menu bar in browse mode.
Browse mode is intended for reading a document, not for interacting with an application.

NVDA will automatically swap to focus mode in the following case:
1. Press down arrow until NVDA reports:
   ```
   Mythical University  navigation landmark    Mythical University  menu bar    menu item  subMenu    About  menu item
   subMenu    Admissions  menu item  subMenu    Academics
   ```
1. Press `alt+down arrow` to expand the menu.
1. Note NVDA changes to focus mode and reports:
   ```
   expanded
   Overview  1 of 4
   ```

Then menu items and submenu items can be explored with arrow keys, the item number is read.

From NVDA speech viewer:
```
out of list  heading    level 2  Example
Mythical University  navigation landmark    Mythical University  menu bar    menu item  subMenu    About  menu item  subMenu    Admissions  menu item  subMenu    Academics
expanded
Overview  1 of 4
Administration  2 of 4
Facts  subMenu  3 of 4
History  1 of 3
```
